### PR TITLE
Potion of Flight

### PR DESCRIPTION
**Pull request by MMMiracles**
it's a potion that ~~makes you into a special snowflake~~ gives you wings, allowing you to effortlessly float over chasms. lizards need not apply.

notice: may or may not save you from feature freezes

### CODE REVIEW

**Comment by KorPhaeron** [comment]
should probably be a boss prize rather than chest but thanks for doing this, saves me the trouble

**Comment by MMMiracles** [comment]
i think it balances out with the fact wings force-drop backpacks, so it pretty heavily limits your inventory as a miner, even with webbing to hold your satchels n' stuff.

**Comment by PKPenguin321** [comment]
>g-guys this feature freeze isn't a total joke i swear

**Comment by MMMiracles** [comment]
refactors necropolis chests

**Comment by Shadowlight213** [comment]
I was not expecting this, although tbh it was inevitable. I am intrigued.

**Comment by TehZombehz** [comment]
I don't quite get the bias against this for lizards but sure, whatever. Can't be hulk, can't have wings. Must be part of the secret agenda to push lizards out. :^)

**Comment by Jazaen** [comment]
I think it's because you'd have to make a whole new mutantrace for them if they were to have angel wings, and you can't really distinguish hulked lizard from a normal green one.

**Comment by Ausops** [comment]
winged lizards are called dragons

we already have dragons and their purpose is to be killed

**Comment by callanrocks** [comment]
Someone wings an organ so we can go further down the snowflake hole in future. Or have a shitty angels vs demons.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -407,3 +407,35 @@
...
+					playsound(loc, 'sound/items/drink.ogg', 50, 1, -1)
+					src.used = 1
+				else
+					C << "<span class='userdanger'>You down the elixir, a terrible pain travels down your back as wings burst out!</span>"
+					C.set_species(/datum/species/angel)
+					playsound(loc, 'sound/items/drink.ogg', 50, 1, -1)
+					playsound(loc, 'sound/items/poster_ripped.ogg', 50, 1, -1)
+					C.adjustBruteLoss(20)
+					C.emote("scream")
+					src.used = 1
```
Comment (phil235) [inline code/modules/mining/lavaland/necropolis_chests.dm]: you can avoid having the drink sound and "used = 1" twice, by putting those lines at the end.

Comment (KorPhaeron) [inline code/modules/mining/lavaland/necropolis_chests.dm]: @mmmiracles fix so I can merge

**Comment by MMMiracles** [comment]
@KorPhaeron i made 2 bugfixes this counts as my first meme pr

**Comment by KorPhaeron** [comment]
This was made before the the PERMAFROST anyway

**Comment by KorPhaeron** [comment]
I'd still rather this be dragon loot than tendril loot though since it gives you max movespeed

**Comment by MMMiracles** [comment]
tendrils really need some better loot to pull from, though.

i mean its not like it can end up like the cursed shovel.

**Comment by KorPhaeron** [comment]
That's true

**Comment by Durkel** [comment]
Holy shit, I used one of these and turned into a winged faggot and dropped my backpack. 0/10, wouldn't drink. Also max run speed doesn't mean shit when viro pumps out a stim virus constantly

**Pull request merged**